sys-kernel/gentoo-kernel-bin/gentoo-kernel-bin-6.6.5.ebuild

```diff
@@ -20,25 +20,130 @@ SRC_URI+="
+# BSD for sys-apps/gentoo-systemd-integration, sys-apps/pcsc-lite
+# BSD for sys-libs/libxcrypt, sys-libs/pam
+# BSD-2 for dev-libs/oniguruma, sys-apps/less, app-crypt/libmd
+# BSD-2 for app-crypt/tpm2-tss, dev-libs/libbsd, sys-libs/libxcrypt
+# BSD-2 for app-emulation/qemu
+# BSD-4 for net-libs/libtirpc, dev-libs/libbsd
+# CC0-1.0 for various app-alternatives symlinks
+# GPL-2 for kernel (modules), dracut and various other files
+# GPL-2 for app-arch/zstd, net-fs/nfs-utils, sys-apps/baselayout
+# GPL-2 for sys-apps/iproute2, sys-apps/kbd, sys-apps/nvme-cli, sys-fs/mdadm
+# GPL-2 for sys-boot/plymouth, sys-fs/btrfs-progs,sys-fs/e2fsprogs
+# GPL-2 for sys-libs/libcap, sys-apps/dbus, dev-libs/libgpg-error, dev-libs/nspr
+# GPL-2 for dev-libs/nss, net-wireless/iwd, sys-apps/keyutils, sys-libs/pam
+# GPL-2 for sys-apps/pcsc-lite, sys-apps/rng-tools, sys-fs/lvm2, sys-fs/dmraid
+# GPL-2 for app-emulation/qemu, sys-apps/net-tools, sys-block/nbd
+# GPL-2 for sys-fs/multipath-tools
+# GPL-2+ for app-arch/xz-utils, dev-libs/lzo, sys-apps/hwdata, sys-apps/nvme-cli
+# GPL-2+ for sys-process/procps, net-misc/networkmanager, sys-fs/cryptsetup
+# GPL-2+ for sys-block/open-iscsi
+# GPL-3 for net-fs/cifs-utils, sys-apps/less, dev-libs/libassuan, net-fs/samba
+# GPL-3+ for gawk, gzip, grep, bash, sed, readline, sys-apps/coreutils
+# GPL-3+ for app-crypt/gnupg, sys-apps/pcsc-lite, sys-libs/readline
+# intel-ucode for Intel microcode/firmware from sys-firmware/intel-microcode
+# ISC for dev-libs/libbsd, net-dns/c-ares, net-misc/curl, sys-apps/pcsc-lite
+# libpng2 for media-libs/libpng
+# linux-fw-redistributable for linux-firmware
+# LGPL-2 for sys-apps/kmod, dev-libs/libaio, app-emulation/qemu
+# LGPL-2+ for sys-process/procps
+# LGPL-2.1 for net-libs/libmnl, sys-apps/acl, sys-apps/attr, sys-libs/libcap-ng
+# LGPL-2.1 for dev-libs/libassuan, dev-libs/libgcrypt, dev-libs/libgpg-error
+# LGPL-2.1 for dev-libs/nss, sys-apps/keyutils, sys-libs/libseccomp
+# LGPL-2.1+ for app-arch/xz-utils, net-libs/libtirpc, sys-libs/libnvme
+# LGPL-2.1+ for sys-process/procps, net-misc/networkmanager, dev-libs/glib
+# LGPL_2.1+ for dev-libs/libtasn1, net-libs/libndp, sys-libs/libxcrypt
+# LGPL-3+ for sys-block/open-iscsi
+# MIT for AMD microcode/firmware from sys-kernel/linux-firmware
+# MIT for app-misc/jq, dev-libs/json-c, sys-libs/ncurses,x11-libs/libdrm
+# MIT for app-crypt/p11-kit, dev-libs/expat, dev-libs/libbsd, dev-libs/libffi
+# MIT for dev-libs/libgcrypt, net-dns/c-ares, sys-apps/pcsc-lite
+# NVIDIA-r1 for nvidia firmware from sys-kernel/linux-firmware
+# ZLIB for sys-libs/zlib
+#
+# UKI only:
+# LGPL-2.1 for systemd-stub from sys-apps/systemd
+# CC-BY-SA-2.5 for our own logo (boot splash)
+LICENSE="
+	GPL-2
+	prebuilt-initramfs? (
+		amd64? ( intel-ucode )
+		x86? ( intel-ucode )
+		Apache-2.0 BEER-WARE BSD BSD-2 BSD-4 curl CC0-1.0 GPL-2 GPL-2+ GPL-3
+		GPL-3+ ISC libpng2 linux-fw-redistributable LGPL-2 LGPL-2+ LGPL-2.1
+		LGPL-2.1+ LGPL-3+ MIT NVIDIA-r1 ZLIB
```

Comment:
Looks like a candidate for a helper variable. I suppose you could use multiple `+=` blocks to avoid having to intersperse licenses and package names.

Reply:
Helper variables are not allowed in LICENSE, pkgcheck complains.